Menu/Menu.js

```diff
@@ -1,10 +1,84 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+
+  if (!menuState) {
+
+    menu.classList.toggle('menu--open');
+    TweenMax.to(menu, 1, {css:{left: '0px'}, ease: Power2.easeInOut});
+
+  }
+
+  else {
+
+    TweenMax.to(menu, 1, {css:{left: '-350px'}, ease: Power2.easeInOut});
+    setTimeout(() => menu.classList.toggle('menu--open'), 1000);
```

Comment:
Another option (other than `setTimeout`) is using [onComplete](https://codepen.io/ChrisLang/pen/VYMgjr?editors=1010). For example:
```javascript
TweenMax.to(menu, 1, {
  css: { left: '-350px' },
  ease: Power2.easeInOut,
  onComplete: () => menu.classList.toggle('menu--open')
});
```